seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSourceConfig.java

```diff
@@ -40,8 +40,8 @@ public class BaseSourceConfig {
                     .noDefaultValue()
                     .withDescription("The file path of source files");
 
-    public static final Option<String> DELIMITER =
-            Options.key("delimiter")
+    public static final Option<String> FIELD_DELIMITER =
```

Comment:
Directly modifying parameter names can cause compatibility issues.
You can update option like this:
```
    public static final Option<String> FIELD_DELIMITER =
            Options.key("field_delimiter")
                    .stringType()
                    .defaultValue(TextFormatConstant.SEPARATOR[0])
                    .withFallbackKeys("delimiter")
                    .withDescription(
                            "The separator between columns in a row of data. Only needed by `text` file format");


```

And then you need update the code at the point used this option.

```
String delimiter =
                        ReadonlyConfig.fromConfig(pluginConfig)
                                .get(BaseSourceConfig.FIELD_DELIMITER);
```